### PR TITLE
EVG-17643 remove unused perf routes, commands, and functions

### PR DESCRIPTION
**Pull request by ablack12**
[EVG-17643](https://jira.mongodb.org/browse/EVG-17643)

### Description 
Remove unused perf routes and the functions they use, as well as two Evergreen commands that are undocumented and unused.

### Testing 
Will verify existing testing works. Local data loads normally. Verified no configs use json.get or json.get_history.

### CODE REVIEW

File: service/ui.go [hunk 1]
```diff
@@ -436,16 +436,6 @@ func (uis *UIServer) GetServiceApp() *gimlet.APIApp {
 
 	// Plugin routes
 	app.PrefixRoute("/plugin").Route("/manifest/get/{project_id}/{revision}").Wrap(needsLogin, viewTasks).Handler(uis.GetManifest).Get()
-	app.PrefixRoute("/plugin").Route("/json/version").Handler(perfGetVersion).Get()
-	app.PrefixRoute("/plugin").Route("/json/version/{version_id}/{name}").Wrap(needsLogin, viewTasks).Handler(perfGetTasksForVersion).Get()
-	app.PrefixRoute("/plugin").Route("/json/version/latest/{project_id}/{name}").Wrap(needsLogin, viewTasks).Handler(perfGetTasksForLatestVersion).Get()
-	app.PrefixRoute("/plugin").Route("/json/task/{task_id}/{name}/").Wrap(needsLogin, viewTasks).Handler(perfGetTaskById).Get()
```
Comment (Kimchelly) [inline service/ui.go:442]: It seems like [`/plugin/json/task`](https://mongodb.splunkcloud.com/en-US/app/search/search?q=search%20index%3Devergreen%20%2Fplugin%2Fjson%2Ftask%20AND%20method%3DGET%20AND%20action%3Dcompleted%20AND%20status!%3D404%20AND%20user&display.page.search.mode=verbose&dispatch.sample_ratio=1&workload_pool=standard_perf&earliest=1663819200&latest=1663905600&sid=1663946775.3897658) and [`/plugin/json/history/{task_id}/{name}`](https://mongodb.splunkcloud.com/en-US/app/search/search?q=search%20index%3Devergreen%20%2Fplugin%2Fjson%2Fhistory%20AND%20method%3DGET%20AND%20action%3Dcompleted%20AND%20status!%3D404%20AND%20user&display.page.search.mode=verbose&dispatch.sample_ratio=1&workload_pool=standard_perf&earliest=1663819200&latest=1663905600&sid=1663946746.3897611) are still being used somewhere according to Splunk. Is this of concern at all?

Comment (ablack12) [reply service/ui.go:442]: Nah these were removed from the UI in yesterday's deploy which is why the usage peters out (the calls that happened post deploy are due to unclosed tabs but the plugin isn't being used anywhere anyway so).

**Review by julianedwards — APPROVED** [review]

**Review by Kimchelly — APPROVED** [review]

**Comment by Kimchelly** [comment]
LGTM mod smoke tests passing.

**Comment by ablack12** [comment]
Will remove json.get_history in a future commit just to be super careful about the smoke test.

**Pull request merged**